server/pbench/bin/unittests

```diff
@@ -105,7 +105,8 @@ function _dump_logs {
 function _verify_output {
     res=$1
     tname=$2
-    diff -cw $_tdir/gold/${tname}.txt $_testout
+    #diff -cw $_tdir/gold/${tname}.txt $_testout
+    cat $_testout | tr '\140' '\047' | diff -cw $_tdir/gold/${tname}.txt -
```

Comment:
Can we keep the `diff` command as is, and transform the test output into a new file?  When unit tests break, it is nice to the original output file to work with.